pkg/config/config.go

```diff
@@ -1029,6 +1029,10 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("admission_controller.auto_instrumentation.patcher.enabled", false)
 	config.BindEnvAndSetDefault("admission_controller.auto_instrumentation.patcher.fallback_to_file_provider", false)                                // to be enabled only in e2e tests
 	config.BindEnvAndSetDefault("admission_controller.auto_instrumentation.patcher.file_provider_path", "/etc/datadog-agent/patch/auto-instru.json") // to be used only in e2e tests
+	config.BindEnv("admission_controller.auto_instrumentation.init_resources.cpu.request")
+	config.BindEnv("admission_controller.auto_instrumentation.init_resources.cpu.limit")
+	config.BindEnv("admission_controller.auto_instrumentation.init_resources.memory.request")
+	config.BindEnv("admission_controller.auto_instrumentation.init_resources.memory.limit")
```

Comment:
maybe we can just set request==limit in that case. because it should be small enough